config/default.json

```diff
@@ -34,9 +34,9 @@
         "EnableUserAccessTokens": false,
         "AllowCorsFrom": "",
         "AllowCookiesForSubdomains": false,
-        "SessionLengthWebInDays": 30,
-        "SessionLengthMobileInDays": 30,
-        "SessionLengthSSOInDays": 30,
+        "SessionLengthWebInDays": 365,
```

Comment:
I think we have to update `model/config.go` too?

https://github.com/mattermost/mattermost-server/blob/master/model/config.go#L376-L386

Reply:
@lieut-data You're right thanks.